docs/flow/modin/backends/pandas/parsers.rst

```diff
@@ -0,0 +1,41 @@
+:orphan:
+
+Pandas Parsers Module Description
+"""""""""""""""""""""""""""""""""
+High-Level Module Overview
+''''''''''''''''''''''''''
+
+This module houses parser classes (classes that are used for data parsing on the workers)
+and util functions for handling parsing results. ``PandasParser`` is base class for parser
+classes with pandas backend, that contains methods common for all child classes. Other
+module classes implement ``parse`` function that performs parsing of specific format data
+basing on the chunk information computed in the ``modin.engines.base.io`` module. After
+chunk data parsing is completed, resulting ``DataFrame``-s will be splitted into smaller
+``DataFrame``-s according to ``num_splits`` parameter, data type and number or
+rows/columns in the parsed chunk, and then these frames and some additional metadata will
+be returned.
+
+Data Parsing Mechanism
+''''''''''''''''''''''
+
+Data parsing mechanism differs depending on the data format type:
+
+..
+  TODO: add link to internal data storage implementation docs to the `text format type section`
+  after DOCS-#2954 is merged
+
+* text format type - file parsing begins from retrieving ``start`` and ``end`` parameters
```

Comment:
This bullet format is a bit tough to read. I don't think we need to document what the code is doing for these, rather what the code expects as input and what it outputs.

Reply:
Ok, i see. Clause with data parsing mechanism was moved to the source code.